core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/heuristic/selector/move/generic/CompositeRuinMoveSelector.java

```diff
@@ -0,0 +1,266 @@
+package org.optaplanner.core.impl.heuristic.selector.move.generic;
+
+import static org.optaplanner.core.impl.heuristic.move.CompositeMove.buildMove;
+
+import java.util.*;
```

Comment:
Asterisk imports are disallowed by our code style. Please configure your IDE not to generate them and look throughout your PR for all instances of this anti-pattern.